### PR TITLE
TSP added in C/C++, Java

### PR DESCRIPTION
**Pull request by dldisha**

### CODE REVIEW

**Comment by jainaman224** [comment]
@dldisha Go ahead.

**Comment by jainaman224** [comment]
@dldisha Add Java in PR description and update the issue with languages you are working on. I can see there is an empty file. Please remove it as well.

File: Graph_coloring_algo/Graph_coloring_algo.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+            for (int j = 0; j < V; j++)
+                graph[i][j] = scan.nextInt();
+ 
+        System.out.println("\nEnter number of colors");
+        int c = scan.nextInt();
+ 
+        gc.graphColor(graph, c);
+ 
+    }
+}
```
Comment (jainaman224) [inline Graph_coloring_algo/Graph_coloring_algo.java]: It is advised to add new line at the end of each file.

Comment (dldisha) [reply Graph_coloring_algo/Graph_coloring_algo.java]: Got it!!

Comment (dldisha) [reply Graph_coloring_algo/Graph_coloring_algo.java]: changes have one #3ea9836

Comment (dldisha) [reply Graph_coloring_algo/Graph_coloring_algo.java]: changes made in thecommit @3ea98366ac039426fa54ebbaa5affa33783fc031

**Comment by jainaman224** [comment]
Please remove graph coloring algorithm files then it is ready to be merged.

**Comment by jainaman224** [comment]
@dldisha You need to create two separate PR for two different issues.

File: Cycle Sort/cycle_sort.cpp [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  
+    sort(a, size);  
+    cout<<"After sort, array : "<<endl;  
+    for (int i = 0; i < size; i++)  
+        cout<<a[i]<<" ";  
+    return 0;  
+} 
+
+// output:
+Enter number of elemnts
```
Comment (jainaman224) [inline Cycle Sort/cycle_sort.cpp]: Comment these outputs.

**Comment by jainaman224** [comment]
Not fixed.

**Comment by jainaman224** [comment]
@dldisha Are you working on it?

**Comment by jainaman224** [comment]
Can I close this PR?

**Comment by dldisha** [comment]
Already done this in another PR #1035. You can close this

**Pull request closed without merging**